app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,84 @@
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all.order(:id)
+  end
+
+  def show
+    @task = Task.find(params[:id].to_i)
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(task_params)
+
+    # (name: params[:task][:name], description: params[:task][:description])
+
+    if @task.save
```

Comment:
When I make a new task, I encounter an error: `completed` on `@task` is nil at this point, and your table requires `completed` to be not nil.

One thing you can do is set `completed` on `@task` before you `.save`
Another thing you can do is make a migration to make `completed` accept nil in the tasks table